docs/configuration.md

```diff
@@ -64,5 +64,26 @@ the default retention for newly created keys that do not have a an override.
 #### Example
 
 ```
-$ redis-server --loadmodule ./redistimeseries.so 20
+$ redis-server --loadmodule ./redistimeseries.so RETENTION_POLICY 20
+```
+
+### DUPLICATE_POLICY
+
+Policy that will define handling of duplicate samples.
+The following are the possible policies:
+* BlOCK
+* FIRST
+* LAST
+
+#### Precedence order
+Since the duplication policy can be provided at different levels, the actual precedence of the used policy will be:
```

Comment:
This should add something about the defaults
as in 
Module configuration (or the default)